utils/docker/Dockerfile-mockbuild.centos:7

```diff
@@ -0,0 +1,69 @@
+#
```

Comment:
Filename not portable to all systems used for development.

Reply:
Fixed.